yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -240,19 +226,19 @@ def test_draw(self):
                 Action.new_occupy(Cell(2, 0)),
                 Action.new_occupy(Cell(0, 1)),
                 Action.new_occupy(Cell(2, 2))])
+        logic = Logic([act_queue_px, act_queue_po])
+        for i in range(0, len(act_queue_px.actions) + len(act_queue_po.actions)):
+            logic.advance(state)
         expected_board_cells = [
                 [Mark.X, Mark.O, Mark.X],
                 [Mark.X, Mark.O, Mark.O],
                 [Mark.O, Mark.X, Mark.O]]
-        logic = Logic([act_queue_px, act_queue_po])
-        for i in range(0, len(act_queue_px.actions) + len(act_queue_po.actions)):
-            logic.advance(state)
-        self.assertEqual(state.board.cells, expected_board_cells)
-        self.assertEqual(state.step, 8)
-        self.assertEqual(state.round, 1)
-        self.assertEqual(state.phase, Phase.OUTROUND)
-        self.assertEqual(state.players[0].wins, 0)
-        self.assertEqual(state.players[1].wins, 0)
+        expected_state = _new_state(
+                board=Board(expected_board_cells),
+                step=8,
+                round_=1,
+                phase=Phase.OUTROUND)
```

Comment:
Is it a requirement to not have spaces around `=` when passing named args?

Reply:
Yes, it is. From [PEP8](https://www.python.org/dev/peps/pep-0008/#:~:text=Don%27t%20use%20spaces%20around%20the%20%3D%20sign%20when%20used%20to%20indicate%20a%20keyword%20argument%2C%20or%20when%20used%20to%20indicate%20a%20default%20value%20for%20an%20unannotated%20function%20parameter):
> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter.